src/js/utilities.ts

```diff
@@ -55,3 +82,14 @@ export function renderClearButton(moduleId: ModuleId) {
 	clearButton.className = "btn btn-sm btn-outline-secondary float-end";
 	header.appendChild(clearButton);
 }
+
+function sentenceCase (str: string) : string {
+    if ((str===null) || (str===''))
+        return "";
+    else
+    str = str.toString();
+     
+    return str.replace(/\w\S*/g,
+    function(txt){return txt.charAt(0).toUpperCase() +
+        txt.substring(1).toLowerCase();});
```

Comment:
But they are already defined in sentence case, so only ToLower would be needed, yeah?